app/assets/javascripts/reducers/stories.js

```diff
@@ -24,6 +24,11 @@ const storiesReducer = (state = initialState, action) => {
         updateIfSameId(action.story.id, (story) => {
           return updateStory(story, action.story);
         }));
+    case actionTypes.SET_LOADING:
+      return state.map(
+        updateIfSameId(action.id, (story) => {
```

Comment:
You can change this to:

```suggestion
        updateIfSameId(action.id, setLoading)
```
since the same param from map goes to `setLoading`.